packages/cli/src/commands/configure/configure_profile_command.test.ts

```diff
@@ -6,27 +6,35 @@ import { ConfigureProfileCommand } from './configure_profile_command.js';
 import { AmplifyPrompter, printer } from '@aws-amplify/cli-core';
 import { Open } from '../open/open.js';
 import { ProfileController } from './profile_controller.js';
+import { UsageDataEmitterFactory } from '@aws-amplify/platform-core';
 
 const testAccessKeyId = 'testAccessKeyId';
 const testSecretAccessKey = 'testSecretAccessKey';
 const testProfile = 'testProfile';
 const testRegion = 'testRegion';
+const usageDataEmitter = await new UsageDataEmitterFactory().getInstance('');
 
 void describe('configure command', () => {
   const profileController = new ProfileController();
+
   const mockAppendAWSFiles = mock.method(
     profileController,
     'createOrAppendAWSFiles',
     () => Promise.resolve()
   );
 
+  const emitFailureSpy = mock.method(usageDataEmitter, 'emitFailure');
+  const emitSuccessSpy = mock.method(usageDataEmitter, 'emitSuccess');
```

Comment:
All commands tests should be updated similarly.

Reply:
The emitter is not attached at the command level. Instead of testing it at unit test level, we should cover this at the integration test.